versions-api/src/test/java/org/codehaus/mojo/versions/ordering/MavenVersionComparatorTest.java

```diff
@@ -39,32 +40,32 @@ public MavenVersionComparatorTest()
     @Test
     public void testSegmentCounting()
     {
-        assertThat( 3, is( instance.getSegmentCount( version( "5" ) ) ) );
-        assertThat( 3, is( instance.getSegmentCount( version( "5.0" ) ) ) );
-        assertThat( 4, is( instance.getSegmentCount( version( "5-0" ) ) ) );
-        assertThat( 1, is( instance.getSegmentCount( version( "5.3.a" ) ) ) );
-        assertThat( 1, is( instance.getSegmentCount( version( "5.0.a.1.4.5" ) ) ) );
-        assertThat( 3, is( instance.getSegmentCount( version( "" ) ) ) );
+        MatcherAssert.assertThat( 3, Is.is( instance.getSegmentCount( version( "5" ) ) ) );
+        MatcherAssert.assertThat( 3, Is.is( instance.getSegmentCount( version( "5.0" ) ) ) );
```

Comment:
I'm not sure if such change is needed for this PR - only make PR longer ...

Reply:
Ok, I'll adjust that tomorrow.